app/services/sockethub-irc.js

```diff
@@ -124,7 +124,7 @@ export default class SockethubIrcService extends Service {
     channel.addUser(account.nickname);
     channel.connected = true;
 
-    channel.addUser(message.actor.name);
+    channel.addUser(message.actor.name || message.actor.id);
```

Comment:
The ID would include the service domain, so that should not be shown in a channel. What's the reason for this fallback? Wouldn't it be a bug if the nickname weren't available as `name`?

Reply:
OK, thanks. I changed it to extract the nickname from the ID when applicable.